owasp-dependency-check-suppressions.xml

```diff
@@ -326,12 +326,16 @@
      <packageUrl regex="true">^pkg:maven/org\.apache\.hadoop/hadoop\-.*@.*$</packageUrl>
      <cve>CVE-2018-11765</cve>
      <cve>CVE-2020-9492</cve>
+     <cve>CVE-2021-31684</cve>
```

Comment:
Shouldnt this go in hadoop 3.3.1 section ?

Reply:
Sorry, just noticed this as well

Reply:
done